index.html

```diff
@@ -0,0 +1,151 @@
+        <ul id="nav_ul">
+            <li><a href="./index.html" >Home</a></li>
+        <li class="about"><a href="./mobile.html">About</a></li>
+        <li><a href="#program">Program</a></li>
+        </ul>
+    </nav>  
+    <main class="principal">
+        <div>
+            <i id="menu-icon" class="fa-solid fa-bars" onclick="showNav()"></i>
+            <i id="menu-icon2" class="fa-solid fa-xmark" onclick="HideNav()"></i>
+        </div>
+        <div class="main">
+            <div class="title">
+                <i>"Bad boy but just good for her"</i>
+                <h1>From a nerd to a straight Stud</h1>
+            </div>
+            <div class="intro">
+                <p>Articles and Videos on Natural Seduction, Lifestyle and Personal Development: Techniques, tips, advice etc.</p>
+            </div>
+            <div class="timeZone">
+                <h1>2022.10.15(THU) ~  16(FRI)</h1>
+                <p class="p3">@central Park, Meet us there and then for a personnal coaching.</p>
+            </div>
+        </div>
+    </main>
+    <section class="pgrm-section" id="program">
+        <div class="main-pgrm">
+            <h2 class="pgrm-title">Main Program</h2>
+            <span class="pgrm-border">program-bro</span>
+        </div>
+        <div class="pgrm-option">
+            <div class="pgrm">
+               <img class="pn1" src="./icon/coaching.png" alt="">
+                <h2 class="sub-lect">Coaching</h2>
+                <p>Listen to  the coaches from various countries
+                     about the messages of having self confidence.</p>
+            </div>
+            <div class="pgrm">
+                <img class="pn2" src="./icon/presentation.png" alt="">
+                <h2 class="sub-exhb">Exhibition</h2>
+                <p>appreciate various amazing books from our mentors about personnal Development.</p>
+            </div>
+            <div class="pgrm">
+                <img class="pn3" src="./icon/forum.png" alt="">
+                <h2 class="sub-frm">Forum</h2>
+                <p>Have your time to share  your thoughts and
+                  opinions with experts for each topic.</p>
+            </div>
+            <div class="pgrm">
+                <img class="pn4" src="./icon/practice.png" alt="">
+                 <h2 class="sub-wshp">Practice</h2>
+                <p>Try to practice by talking to some strangers rather than just staying in your comfort-zone.</p>
+            </div>
+            <div class="pgrm">
+                <img class="pn4" src="./icon/ignite.png" alt="">
+                <h2 class="sub-ignt">Ignite</h2>
+                <p>Get opportunities of being selected for a vacation in the Maldive Islands.</p>
+            </div>
+        </div>
+        <div class="pgrm-footer">
+            <h2 class="pgrm_btn"> <a href="" >SEE THE WHOLE PROGRAM</a> </h2>
+            <h2 class="pgrm_btn2"> <a href="" >Join Ideal Man Community 2022</a> </h2>
```

Comment:
- [ ] Great job so far, here to be more semantic you can use a button instead of a header to wrap the anchor element, and also the text should be centred and a one-liner, kindly fix that.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/163626128-c6f10d78-89b5-42b8-85ad-4b67a6f95540.png)

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/163626170-ec22d65c-5ebf-4ad3-8bc9-125cd238c42c.png)